### PR TITLE
Email notifications

### PR DESCRIPTION
**Pull request by AvidDabbler**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

# What's new in this PR

- added saving file function
- pull emails from airtable
- saving missing stores and products to root directory
- created cli function for scheduled send of missing products and stores to email list in airtable

[//]: # "Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

Responding to client on how to easily send out an email with missing products and stores in comparison between appian and airtable.

### CODE REVIEW

**Comment by AvidDabbler** [comment]
@ziveo are we good on this?

File: utils/appian.js [hunk 1]
```diff
@@ -66,6 +69,8 @@ export default async function getProducts() {
     throw new Error(`Appian API response status: ${response.status}`);
   }
   const { data } = await response.json();
+  saveFile('./data/storesData.json', data);
+  console.log('appian data: ', data);
```
Comment (defiantgoat) [inline utils/appian.js:73]: do you need the console.log?

File: send-email.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+const sendEmail = async () => {
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    auth: {
+      user: process.env.EMAIL_USER,
+      pass: process.env.EMAIL_PASS,
+    },
+  });
+
+  const emailList = await getAllRecords(Tables.Emails)
```
Comment (defiantgoat) [inline send-email.js]: you could probably do away with the .then block if you are using await.

**Review by defiantgoat — APPROVED** [review]
Just made a few comments

File: utils/appian.js (hunk 1 above)
Comment (AvidDabbler) [reply utils/appian.js:73]: We have console logs in here to match the debugging that included. we use this information to debug service logs on heroku

File: send-email.js (hunk 2 above)
Comment (AvidDabbler) [reply send-email.js]: removed await

**Pull request merged**